content/community/workshops/precice-workshop-2027.md

```diff
@@ -0,0 +1,87 @@
+---
+title: preCICE workshop 2027
+keywords: precice workshop 2027, event, events, workshop
+summary:
+permalink: precice-workshop-2027.html
+toc: true
+redirect_from: /workshop2027/
+---
+
+<img class="img-responsive center-block" src="images/events/precice2027/precice2027-path.svg" alt="preCICE Workshop banner" style="max-width: 500px; width: 100%; margin:auto;">
+
+The 7th preCICE Workshop will be held for the first time outside Germany, at the [Delft University of Technology](https://www.tudelft.nl/en/) in The Netherlands, on March 1-5, 2027, co-organized by the local user group and the [G+Smo community](https://gismo.github.io/). The workshop is a coming together of the preCICE community to share ideas, experiences and knowledge about using preCICE, and to learn from others in the process. Like always, we plan to have user and developer talks, hands-on training sessions, discussions with the developers about your applications and use cases, and plenty of opportunities for networking. Read more about [how a preCICE workshop looks like](precice-workshop.html).
+
+The workshop will include a hands-on [training course](community-training.html). The course is suited for both beginners and current preCICE users, since advanced topics will also be covered. This year, we are extending the course by a new module on best practices in the development of preCICE adapters.
```

Comment:
```suggestion
The workshop will include a hands-on [training course](community-training.html). The course is suited for both beginners and current preCICE users, since advanced topics will also be covered. This year, we are extending the course by two new modules: one on best practices in the development of preCICE adapters and one on how to couple solvers using Isogeometric Analysis (IGA).
```